Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -43,8 +43,6 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
     let viewName: String
     /// The start time of this View.
     let viewStartTime: Date
-    /// Date correction to server time.
-    private let dateCorrection: DateCorrection
```

Comment:
I think this property wasn't enough described in this inline comment. We cannot simply remove local `dateCorrection` from view scope and use its global value from SDK context like proposed. This will lead to skewing events in RUM (🐞).

Note that the result of NTP sync done with Kronos arrives over time. When the app starts, we perform the sync and it takes up to several seconds until the final result is received. Meantime the SDK receives intermediate results - each next with better precision. Because `DateCorrection` abstracts the Kronos sync, its values cannot be considered stable or monotonic.

The idea of capturing `dateCorrection` in view scope's property is:
* to "freeze" the (NTP) date correction value when the view scope is created;
* to apply the same, "frozen" correction to all events sent from view scope;
* to pass the same ("frozen") date correction value to all child scopes (e.g. resource scope) for applying it to events sent from nested scopes.

If we don't "freeze" date correction and instead always use the one from SDK context, it means we will be applying different deltas to view scope events and its child scope events, leading to noticeable skews.